components/kyma-environment-broker/README.md

```diff
@@ -2,21 +2,33 @@
 
 ## Overview
 
-Kyma Environment Broker provides a way to run Kyma as a Runtime on clusters provided by different cloud providers, such as AWS, GCP or Azure. The broker provides services that install Kyma with a separate plan for each provider.
-It uses the Compass Provisioner API to install Kyma on the given cluster.
+Kyma Environment Broker is a component that allows you to run Kyma as a Runtime on clusters provided by third-party providers. It uses Provisioner's API to install Kyma on a given cluster.
+
+For more information, read the [documentation](../../docs/kyma-environment-broker).
+
 
 ## Development
 
-Use the following environment variables to configure the Kyma Environment Broker:
+This table lists the environment variables, their descriptions, and default values:
 
-| Name | Required | Default | Description |
-|-----|:---------:|--------|------------|
-| **APP_PORT** | No | `8080` | The port on which the HTTP server listens. |
-| **APP_PROVISIONING_URL** | No |  | Specifies an URL to the provisioner API. |
-| **APP_PROVISIONING_SECRET_NAME** | No | | Specifies the name of the Secret which holds credentials to the Provisioner API. |
-| **APP_PROVISIONING_GARDENER_PROJECT_NAME** | No | `true` | Defines the used Gardener project name. |
-| **APP_PROVISIONING_GCP_SECRET_NAME** | No | | Defines the name of the Secret which holds credentials to GCP. |
-| **APP_PROVISIONING_AWS_SECRET_NAME** | No | | Defines the name of the Secret which holds credentials to AWS. |
-| **APP_PROVISIONING_AZURE_SECRET_NAME** | No | | Defines the name of the Secret which holds credentials to Azure. |
-| **APP_AUTH_USERNAME** | No | | Specifies the Kyma Environment Service Broker authentication username. |
-| **APP_AUTH_PASSWORD** | No | | Specifies the Kyma Environment Service Broker authentication password. |
+| Name | Description | Default value |
+|-----|---------|:--------:|
+| **APP_PORT** | The port on which the HTTP server listens. | `8080` |
```

Comment:
I would be consistent in descriptions and start all wither with a noun or with a verb